src/main/java/org/iac2/service/architecturereconstruction/plugin/implementation/opentoscacontainer/OpenToscaContainerModelCreationPlugin.java

```diff
@@ -48,14 +49,7 @@ public class OpenToscaContainerModelCreationPlugin implements ModelCreationPlugi
     private final static int OPENTOSCA_CLIENT_TIMEOUT = 10000;
 
     public OpenToscaContainerModelCreationPlugin() {
-        EdmmTypeResolver.putMapping("docker_engine", DockerEngine.class);
-        EdmmTypeResolver.putMapping("docker_container", DockerContainer.class);
-        EdmmTypeResolver.putMapping("mysql_dbms", MySqlDbms.class);
-        EdmmTypeResolver.putMapping("mysql_db", MySqlDb.class);
-        EdmmTypeResolver.putMapping("realworld_application_backend_java11_spring", RealWorldApplicationBackendJava11Spring.class);
-        EdmmTypeResolver.putMapping("java_11", Java11.class);
-        EdmmTypeResolver.putMapping("realworld_application_angular", RealWorldAngularApp.class);
-        EdmmTypeResolver.putMapping("nginx", Nginx.class);
+        EdmmTypeResolver.initDefaultMappings();
```

Comment:
Good idea.. I will try to use a bean for this purpose

Reply:
I moved it to the `CommandLineRunner` bean. However, this will only run if the spring boot context is loaded. Therefore, tests that are not annotated with @SpringBootTest will not run it. Adding the annotation everywhere is also a bad idea because loading the context takes time and requires a functional database (docker container)... therefore, only in tests I will leave the `EdmmTypeResolver.initDefaultMappings();` invocation.